setup.py

```diff
@@ -72,7 +74,9 @@
     include_package_data=True,
     install_requires=requires,
     extras_require={
-      "debug": ["rpdb"]
+        "debug": ["rpdb"],
```

Comment:
I think `graphviz` is missing here as an optional dependency.

Reply:
this I don't understand?

Reply:
This is not a comment linked to you changes, but I noticed that `graphviz` is present in the requirements.txt file as an optional dependency (See: https://github.com/inmanta/inmanta-core/blob/8c1dabb9742bdc3800744f16b85f48298cf8df23/requirements.txt#L28). But It's not present in the `extras_require` list. I though would could include it in this PR, since it's a very small change. What do you think?